unlock-app/src/__tests__/helpers/Locks.test.js

```diff
@@ -1,4 +1,4 @@
-import {getLockTransaction, getLockConfirmations, getLockStatusString} from '../../helpers/Locks'
+import {getLockTransaction, getLockConfirmations, getLockStatusString, getLockName} from '../../helpers/Locks'
```

Comment:
Why do you use the helper for that? Why not just get that value from the `lock` object?